test/integ/workflowapi/test_workflow_crud.py

```diff
@@ -0,0 +1,147 @@
+            monitor_lambda = operation_config["Input"]+operation_config["Type"]+operation_config["Status"]+"MonitorLambda"
+            operation_body["MonitorLambdaArn"] = api.stack_resources[monitor_lambda]
+
+        if "OutputMediaType" in operation_config:
+            operation_body["Configuration"]["OutputMediaType"] = operation_config["OutputMediaType"]
+
+        if "TestCustomConfig" in operation_config:
+            operation_body["Configuration"]["TestCustomConfig"] = operation_config["TestCustomConfig"]
+
+        create_operation_response = api.create_operation_request(operation_body)
+        assert create_operation_response.status_code == 200
+        validation.schema(create_operation_response.json(), api_schema["create_operation_response"])
+
+        # Read the operation
+
+        print("Running /workflow/operation API read tests")
+
+        get_operation_response = api.get_operation_request(operation_config["Name"])
+        assert get_operation_response.status_code == 200
+
+    print("Running /workflow/stage API create tests")
+
+    for stage_config in stage_configs:
+        print("\n----------------------------------------")
+        print("\nTEST CONFIGURATION: {}".format(stage_config))
+
+        # Create the stage
+
+        stage_body = {
+            "Name": stage_config["Name"],
+            "Operations": stage_config["Operations"]
+        }
+
+        create_stage_response = api.create_stage_request(stage_body)
+        assert create_stage_response.status_code == 200
+        validation.schema(create_stage_response.json(), api_schema["create_stage_response"])
+
+        # Read the stage
+
+        print("Running /workflow/stage API read tests")
+
+        get_stage_response = api.get_stage_request(stage_config["Name"])
+        assert get_stage_response.status_code == 200
+
+    print("Running /workflow API create tests")
+
+    for workflow_config in workflow_configs:
+        print("\n----------------------------------------")
+        print("\nTEST CONFIGURATION: {}".format(workflow_config))
+
+        workflow_body = {
+            "Name":workflow_config["Name"],
+            "StartAt": stage_configs[0]["Name"]
+        }
+
+        workflow_stages = {}
+        num_stages = len(stage_configs)
+        i = 1
+        for stages in stage_configs:
+            if i == num_stages:
+                workflow_stages[stages["Name"]] = {
+                    "End": True
+                }
+            else:
+                workflow_stages[stages["Name"]] = {
+                    "Next": stage_configs[i]["Name"]
+                }
+            i = i + 1
+
+        workflow_body["Stages"] = workflow_stages
+
+        create_workflow_response = api.create_workflow_request(workflow_body)
+        assert create_workflow_response.status_code == 200
+        # FIXME - need to update schema manually
+        # #validation.schema(workflow, api_schema["create_workflow_response"])
```

Comment:
Is this supposed to use the jsonschema defintions from the workflowapi for the create operation?  Not sure I understand what need to be fixed here.   Maybe this FIXME is just leftover from original code.

Reply:
This is indeed leftover from the previous code. Not sure what this is.